src/ex2/PokemonClient.js

```diff
@@ -11,8 +13,7 @@ class PokemonClient {
 
       return data;
     } catch (error) {
-      console.log(error);
-      return `${pokemonId} Not found`;
+      // console.log(error);
```

Comment:
worth cleaning up things like this
If your catch doesn't do anything then you can just delete it :)